_data/accessibility-tests/prose.yml

```diff
@@ -0,0 +1,37 @@
+title: Prose
+component_status: pass
+test_items:
+# General tests
+  - summary: Information remains usable after style changes.
+    summary_additional: After you adjusts text spacing, you can still access and use all of the information on the page. You can see that content adjusts correctly in response to styling changes.
+    test_status: pass
+    test_type: general
+    version_tested: 3.7.1
+    wcag_criterion: 1.4.12
+  - summary: Content is written in plain language.
+    summary_additional: Content is easy to understand. It is written at the appropriate grade level. Complex words are defined by a glossary or other method.
```

Comment:
@sarah-sch @finekatie 
It is not immediately clear to me what "appropriate grade level" means.  I'm concerned users won't know how to test that. Can we add some specifics here?

Reply:
If we want to send folks to the source we can refer them to [this W3C page](https://www.w3.org/WAI/WCAG21/Understanding/reading-level.html) which has a lot of resources at the bottom.

Reply:
I can update `grade level` to link to that resource. It's a great resource, though _it is_ linked through the WCAG 3.1.5 link right beneath it. I see the benefit there as long as we don't think it's too redundant!  

Let me know what you think

<img width="702" alt="image" src="https://github.com/uswds/uswds-site/assets/25211650/1f403556-5a29-4aed-8939-a6f185046e84">

Reply:
Is there some way to create an anchor link to that section at the bottom?

Reply:
@amycole501 The `3.1.5 Reading Level` button will take users to the same guidance. Do you think that's enough of a "more information here" for people who might have questions?

Reply:
@amyleadem - I think @mahoneycm's solution of linking to more information from "grade level" is a good one.

Reply:
Resolved in [7121fff](https://github.com/uswds/uswds-site/pull/2613/commits/7121fff586c97b3e127cbbab0a42b7a76e63a80e)